pr_agent/tools/pr_description.py

```diff
@@ -73,14 +73,18 @@ async def run(self):
         if get_settings().config.publish_output:
             logging.info('Pushing answer...')
             if get_settings().pr_description.publish_description_as_comment:
-                self.git_provider.publish_comment(markdown_text)
+                self.git_provider.publish_comment(pr_body)
             else:
-                self.git_provider.publish_description(pr_title, pr_body)
-                if self.git_provider.is_supported("get_labels"):
-                    current_labels = self.git_provider.get_labels()
-                    if current_labels is None:
-                        current_labels = []
-                    self.git_provider.publish_labels(pr_types + current_labels)
+                # bitbucket does not support publishing PR labels yet
+                if get_settings().config.git_provider == 'bitbucket':
+                    return 
```

Comment:
**Suggestion:** Consider removing the redundant `return` statement in line 80. If the function doesn't need to return a value, there's no need to include a `return` statement.
```suggestion
                if get_settings().config.git_provider == 'bitbucket':
                    pass
```